docs/proposals/multiple-schedulers.md

```diff
@@ -152,6 +152,11 @@ which the client has set a scheduler annotation that does not correspond to a ru
 2. Dynamic launching scheduler(s) and registering to admission controller (as an external call). This also
 requires some work on authorization and authentication to control what schedulers can write the /binding
 subresource of which pods.
+3. Optimize the behaviors of priority functions in multi-scheduler scenario. Running multi-scheduler would make the
+priority functions related to resource allocation less meaningful/efficient. Since all schedulers would tend to pick
+the node with most free resource available. To solve this problem, We can pass a optional flag such as `randomize-node-selection`
```

Comment:
I would say `randomize-node-selection=N` to make it clearer that the `N` you mention later is the argument to this parameter